crates/bitwarden-crypto/src/error.rs

```diff
@@ -16,8 +17,8 @@ pub enum CryptoError {
     InvalidKeyLen,
     #[error("The value is not a valid UTF8 String")]
     InvalidUtf8String,
-    #[error("Missing Key")]
-    MissingKey,
+    #[error("Missing Key for organization with ID {0}")]
+    MissingKey(Uuid),
```

Comment:
Would it be more appropriate to have `MissingKeyUser`, `MissingKeyOrg(id)` as variants of a MissingKey enum?

Reply:
A user key will never be missing, as `EncryptionSettings` always has it defined. Only case where a user key is not present is when `EncryptionSettings` is `None`, and in those cases we return `VaultLocked`. 

`enc.get_key(&None)` always returns `Ok` with the user key.

We could rename `MissingKey` to `MissingOrgKey` to be more specific though.